modules/ptp-configuring-dynamic-leap-seconds-handling-for-tgm.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_mod-docs-content-type: PROCEDURE
+[id="ptp-configuring-dynamic-leap-seconds-handling-for-tgm_{context}"]
+= Configuring dynamic leap seconds handling for PTP grandmaster clocks
+
+The PTP Operator container image includes the latest `leap-seconds.list` file  that is available at the time of release.
+You can configure the PTP Operator to automatically update the leap second file by using GPS announcements.
+
+Leap second information is stored in an automatically generated `leap-configmap` custom resource (CR) in the `openshift-ptp` namespace.
+The PTP Operator mounts the `leap-configmap` CR as a volume in the `linuxptp-daemon` pod that is accessible by the `ts2phc` process.
+
+If the GPS satellite broadcasts new leap second data, the PTP Operator updates the `leap-configmap` CR with the new data.
```

Comment:
Technically ConfigMap is not a custom resource (CR). Custom resources are only those defined by a CustomResourceDefinition (CRD). But ConfigMap is a **core** resource.

```suggestion
Leap second information is stored in an automatically generated ConfigMap resource named `leap-configmap` in the `openshift-ptp` namespace.
The PTP Operator mounts the `leap-configmap` as a volume in the `linuxptp-daemon` pod that is accessible by the `ts2phc` process.

If the GPS satellite broadcasts new leap second data, the PTP Operator updates the `leap-configmap` with the new data.
```